package.json

```diff
@@ -66,6 +66,7 @@
   "dependencies": {
     "axios": "^1.3.6",
     "jsonwebtoken": "^9.0.0",
+    "pnpm": "^8.3.1",
```

Comment:
This shouldn't be added as a dependency. The correct version is already specified by the `packageManager` key in here.